### ISSUE TITLE
Build failure: redundant imports

### ISSUE THREAD

**Issue by yonas** [dani-garcia/vaultwarden#4476]
```
$ cargo build --features sqlite --release
...
error: the item `jsonwebtoken` is imported redundantly
 --> src/auth.rs:7:20
  |
7 | use jsonwebtoken::{self, errors::ErrorKind, Algorithm, DecodingKey, EncodingKey, Header};
  |                    ^^^^ the item `jsonwebtoken` is already defined by prelude
  |
  = note: `-D unused-imports` implied by `-D unused`
  = help: to override `-D unused` add `#[allow(unused_imports)]`

error: the item `serde_json` is imported redundantly
   --> src/util.rs:561:18
    |
561 | use serde_json::{self, Value};
    |                  ^^^^
    |
   ::: src/main.rs:20:1
    |
20  | extern crate serde_json;
    | ------------------------ the item `serde_json` is already imported here

error: could not compile `vaultwarden` (bin "vaultwarden") due to 2 previous errors
```

**Comment by BlackDex** [comment]
This is already addressed in #4475.

**Comment by BlackDex** [comment]
Also as a side note, this does not happen with stable rust which is what we support as our maximum Rust version.
